src/wallet/rpcwallet.cpp

```diff
@@ -3531,6 +3532,9 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     if (contextualTx.nVersion == 1) {
         contextualTx.nVersion = 2; // Tx format should support vjoinsplits 
     }
+    if (NetworkUpgradeActive(chainActive.Height(), Params().GetConsensus(), Consensus::UPGRADE_OVERWINTER)) {
```

Comment:
`chainActive.Height() + 1` (here and everywhere else, because we care about the next block that will be mined).